File: tests/test-services/java/template/src/main/java/theservice/services/v2/EchoServiceImpl.java
package theservice.services.v2;

import java.math.BigDecimal;
import java.time.*;
import java.util.*;

{{#server.micronaut}}
import io.micronaut.context.annotation.Bean;
{{/server.micronaut}}
{{#server.spring}}
import org.springframework.stereotype.Service;
{{/server.spring}}

import theservice.v2.models.*;
import theservice.v2.services.echo.*;

{{#server.micronaut}}
@Bean
{{/server.micronaut}}
{{#server.spring}}
@Service("EchoServiceV2")
{{/server.spring}}
public class EchoServiceImpl implements EchoService {
	@Override
	public Message echoBodyModel(Message body) {
		return body;
	}
}

File: test-services/kotlin/template/src/main/kotlin/theservice/services/v2/EchoServiceImpl.kt
package theservice.services.v2

{{#server.micronaut}}
import io.micronaut.context.annotation.Bean
{{/server.micronaut}}
{{#server.spring}}
import org.springframework.stereotype.Service
{{/server.spring}}
import theservice.v2.models.*
import theservice.v2.services.echo.*
import java.math.BigDecimal
import java.time.*
import java.util.*
import java.io.*

{{#server.micronaut}}
@Bean
{{/server.micronaut}}
{{#server.spring}}
@Service("EchoServiceV2")
{{/server.spring}}
class EchoServiceImpl : EchoService {
	override fun echoBodyModel(body: Message): Message {
        return body
	}
}
